### DISCUSSION TITLE
Autogen

### DISCUSSION THREAD

**Discussion by evwltrs** [kubilus1/autoortho#48] [General]
Hi,

I am missing autogen in the regions where autoortho is installed. I also noticed I don't have an overlay folder. Any ideas why this didnt appear and how can I get the overlays

Thanks

**Comment by evwltrs** [comment]
@kubilus1 any ideas about this?

**Comment by kubilus1** [comment]
Ahh I forgot to add the overlays zips to the latest package of scenery.   You should see updates shortly.  (If you are using Simheaven, you don't need to update)
